lib/users/index.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
The convention we have in place is to have `lib/thing.js` and `lib/thing/foo.js`, not `lib/thing/index.js`. Please either follow the existing convention, or update the other files to this convention. FWIW this is easier to tab to in the command line